src/eventshub/dist/eventshub.js

```diff
@@ -1,3 +1,4 @@
+(function() {
```

Comment:
Wrap the whole thing in a function enclosure so outside stuff stays outside stuff.